cluster/docker-cluster/configNetwork.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# reconfigure docker network setting
+set -e
+
+source ~/docker-cluster/kube-config/node.env
+
+# TODO exit 1 will break ssh, maybe just exit?
+if [ "$(id -u)" != "0" ]; then
+  echo >&2 "configNetwork must be called as root!"
+  exit 1
+fi
+
+if [[ ! -z $1 ]]; then
+  # On master, we set flannel net config
+  docker -H unix:///var/run/docker-bootstrap.sock run \
+    --net=host gcr.io/google_containers/etcd:$ETCD_VERSION \
+    etcdctl set /coreos.com/network/config \
+    "{ \"Network\": \"$FLANNEL_NET\", \"Backend\": {\"Type\": \"vxlan\"}}"
+
+fi
+
+
+# TODO iface may change to a private network interface, eth0 is for default
+flannelCID=$(docker -H unix:///var/run/docker-bootstrap.sock run \
+    --restart=always -d --net=host --privileged \
+    -v /dev/net:/dev/net quay.io/coreos/flannel:$FLANNEL_VERSION \
+    /opt/bin/flanneld --etcd-endpoints=http://${MASTER_IP}:4001 -iface="eth0")
+
+# sleep to wait network
```

Comment:
In some case node may both have eth0 & eth1, but we use eth0 here as default & non-configurable. I'll update the comment.